File: 000_jupyter_notebooks/stage_fmm_maksim/002_stage_cat_cat_maksim_tomic_day_to_day_2/002_maksim_harcelement.py
# 001_maksim_football_team_rating.py

"""
# chemin
cd /Users/brunoflaven/Documents/03_git/learning_python_for_all/000_jupyter_notebooks/stage_fmm_maksim/002_stage_cat_cat_maksim_tomic_day_to_day_2

# lancer le programme
python 002_maksim_harcelement.py

"""

# print('Dobar Dan Maksim !')

def signaler_harcelement(duree_en_semaines):
    if duree_en_semaines == 1:
        return "Parle-en aux surveillants."
    elif 2 <= duree_en_semaines <= 3:
        return "Parle-en à ton professeur principal."
    elif 6 <= duree_en_semaines <= 8:
        return "Parle-en au proviseur."
    else:
        return "Cherche de l'aide immédiatement auprès d'un adulte de confiance."

# Exemple d'utilisation
duree_en_semaines = int(input("Depuis combien de semaines subis-tu du harcèlement ? "))
print(signaler_harcelement(duree_en_semaines))

File: 000_jupyter_notebooks/stage_fmm_maksim/002_stage_cat_cat_maksim_tomic_day_to_day_2/002_maksim_harcelement.js
/* 001_maksim_football_team_rating.js */ 

function signalerHarcelement(dureeEnSemaines) {
    if (dureeEnSemaines === 1) {
        return "Parle-en aux surveillants.";
    } else if (dureeEnSemaines >= 2 && dureeEnSemaines <= 3) {
        return "Parle-en à ton professeur principal.";
    } else if (dureeEnSemaines >= 6 && dureeEnSemaines <= 8) {
        return "Parle-en au proviseur.";
    } else {
        return "Cherche de l'aide immédiatement auprès d'un adulte de confiance.";
    }
}

// Exemple d'utilisation
const dureeEnSemaines = parseInt(prompt("Depuis combien de semaines subis-tu du harcèlement ? "));
console.log(signalerHarcelement(dureeEnSemaines));


